sdks/python/apache_beam/metrics/metric.py

```diff
@@ -339,8 +339,8 @@ class Lineage:
   for lineage tracking."""
 
   LINEAGE_NAMESPACE = "lineage"
-  SOURCE = "sources"
-  SINK = "sinks"
+  SOURCE = "sources_v2"
```

Comment:
Since metrics are namespaced would it be better to do keep name sources and sinks itself and change the namespace to lineage_v2?

Reply:
The division between the namespace and non-namespace portion of a name are somewhat arbitrary... Keeping the namespace the same is nice because it allows one to query for all lineage counters (even as they evolve) but I don't feel too strongly about this.

Reply:
SG. I agree either way is fine. Lets keep the namespace common then. Thanks.